### PR TITLE
feature(ManagementBackup): support backup to google storage

### PR DESCRIPTION
**Pull request by fruch**
For running and testing the GCE backup with bigger loads,
we want to be able to run this nemesis on GCE too.

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Review by fgelcer — APPROVED** [review]
LGTM

File: sdcm/nemesis.py [hunk 1]
```diff
@@ -1370,16 +1370,23 @@ def disrupt_modify_table(self):
...
         if not self.cluster.params.get('backup_bucket_location'):
             raise UnsupportedNemesis('backup bucket location configuration is not defined!')
 
         mgr_cluster = self.cluster.get_cluster_manager()
-        bucket_location_name = self.cluster.params.get('backup_bucket_location').split()
-        mgr_task = mgr_cluster.create_backup_task(location_list=['s3:{}'.format(bucket_location_name[0])])
+        cluster_backend = self.cluster.params.get('cluster_backend')
+        backup_bucket_location = self.cluster.params.get('backup_bucket_location')
+        if cluster_backend == 'aws':
+            bucket_name = f"s3:{backup_bucket_location.split()[0]}"
```
Comment (yarongilor) [inline sdcm/nemesis.py:1384]: why configuring a parameter value that has to be splited later? why not having the simple wanted value?

File: sdcm/nemesis.py [hunk 2]
```diff
@@ -1370,16 +1370,23 @@ def disrupt_modify_table(self):
...
         if not self.cluster.params.get('use_mgmt', default=None):
             raise UnsupportedNemesis('Scylla-manager configuration is not defined!')
         if not self.cluster.params.get('backup_bucket_location'):
             raise UnsupportedNemesis('backup bucket location configuration is not defined!')
 
         mgr_cluster = self.cluster.get_cluster_manager()
-        bucket_location_name = self.cluster.params.get('backup_bucket_location').split()
-        mgr_task = mgr_cluster.create_backup_task(location_list=['s3:{}'.format(bucket_location_name[0])])
+        cluster_backend = self.cluster.params.get('cluster_backend')
+        backup_bucket_location = self.cluster.params.get('backup_bucket_location')
```
Comment (yarongilor) [inline sdcm/nemesis.py:1382]: in order not to confuse aws and gce bucket location names. perhaps we can consider splitting to 2 parameters with different prefixes like aws_backup_bucket_location and gce_backup_bucket_location

File: sdcm/nemesis.py (hunk 1 above)
Comment (fruch) [reply sdcm/nemesis.py:1384]: Cause it has different structure in AWS, in AWS we have a parameter per region, in GCE we use only one region and don't have this parameters per region structure

File: sdcm/nemesis.py (hunk 2 above)
Comment (fruch) [reply sdcm/nemesis.py:1382]: I don't see the point of adding more parameters, it mean the same thing, and used the same.

File: sdcm/nemesis.py (hunk 1 above)
Comment (fgelcer) [reply sdcm/nemesis.py:1384]: it was introduced this way as in AWS we run (all sanity tests) with multi DC, so we need to pick one bucket, not the whole list, as it has to change the command to use multiple buckets

File: sdcm/nemesis.py (hunk 2 above)
Comment (fgelcer) [reply sdcm/nemesis.py:1382]: i agree with @fruch , as this is exactly same thing... we must avoid creating repeated variables, when they mean the same thing, and used the same way...

**Pull request merged**